src/www/widgets/api/plugins/gateway.inc

```diff
@@ -49,6 +50,9 @@ function gateway_api()
                 case "delay":
                     $gatewayItem['status_translated'] = gettext("Latency");
                     break;
+                case "stdev":
+                    $gatewayItem['status_translated'] = gettext("StDev");
```

Comment:
STDEV

Reply:
Actually it should be RTTd, - That changed in V2 but fell through my review.

Reply:
Fine, it should simply be the same everywhere

Reply:
Or "Standard Derivation"

Reply:
Not Derivation, that's from derived.

Deviation :)

Reply:
oops